src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+import './TodoInfo.scss';
+import { UserInfo } from '../UserInfo/UserInfo';
+
+type Props = {
+  todo: Todo,
+};
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => (
+  <div key={todo.id} className="todo">
+    <UserInfo user={todo.user} />
+    <div className="todo__title">{`Todo: ${todo.title}`}</div>
+    {todo.completed
+      ? <span className="todo__completed">Completed</span>
+      : <span className="todo__completed">Not completed</span>}
```

Comment:
use conditional rendering only for text
```suggestion
    <span className="todo__completed">
      {todo.completed
        ? "Completed"
        : "Not completed"
      }
    </span>
```